### DISCUSSION TITLE
Adding a comment to 'ignored' BPA rules in a model

### DISCUSSION THREAD

**Discussion by WayneGibbings** [TabularEditor/TabularEditor3#1141] [Ideas]
As a peer reviewer, or subsequent model developer, I would like to know why the developer as ignored a BPA rule - there may be valid reasons and it would seem that documenting them against the rule at the time they are ignoring them would be helpful.

What I'm thinking is that when selecting "ignore" on a rule the developer has the option of adding a comment as to why they are ignoring the rule, which is saved in the model when the ignore setting is saved

When viewing ignored rules in BPA, you can surface the comment next to the ignored rule. That way, a developer can pick up the model in six months time, review the model and understand why there are ignored rules

**Comment by mlonsk** [comment]
Hi @WayneGibbings 
Very sorry for not answering this originally, was an oversight of ours.

I completely see the reasoning for your suggestion and will talk to our developers about its feasibility.
